src/main.rs

```diff
@@ -32,7 +34,18 @@ type Hash = Uint256;
 async fn main() -> Result<(), Error> {
     let mut opt: Opt = Opt::parse();
     opt.process()?;
-    env_logger::builder().filter_level(opt.log_level()).parse_default_env().init();
+
+    let mut builder = env_logger::builder();
+    builder.filter_level(opt.log_level()).parse_default_env();
+    if opt.altlogs {
+        builder.format(|buf, record| {
+            let timestamp = Local::now().format("%Y-%m-%d %H:%M:%S%.3f%:z");
+            writeln!(buf, "{} [{:>5}] {}", timestamp, record.level(), record.args())
+        });
+    }
+    builder.init();
```

Comment:
What's/Why's the change here?

Reply:
As you can see in the screenshot below, the desktop application runs kaspad and the miner and sanitizes logs, removing timestamps, which are not needed on the desktop.  It is easier for me to use the same filtering function (+ the native logs have variable prefix lengths, and I wanted fixed length not to do any additional processing).  I did not want to change dependencies to match kaspad, so I added a custom formatting function.

![5436A39A-3E35-48D5-937B-C09467B65039_1_105_c](https://github.com/elichai/kaspa-miner/assets/7544020/7cab5802-fe9a-41f7-9f07-f0ab318ddee5)

Reply:
note where it says: launching 1 cpu miners, Found a block.. etc